setup.py

```diff
@@ -0,0 +1,85 @@
+#   Copyright 2020 Kevin Systrom
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+#!/usr/bin/env python
+from codecs import open
+from os.path import realpath, dirname, join
+from setuptools import setup, find_packages
+import re
+
+DISTNAME = "covid"
```

Comment:
the "covid" package name is already taken on PyPI. Maybe discuss a rename to "rtlive"?